db_client/models/organisation/counters.py

```diff
@@ -60,7 +62,7 @@ class EntityCounter(Base):
     _get_and_increment = text(
         """
         WITH updated AS (
-        UPDATE entity_counter SET counter = counter + 1
+        UPDATE entity_counter SET counter = COALESCE(0, counter) + 1
```

Comment:
should this not be `COALESE(counter, 0)` as I thought it returns the first non null value in the list?

Reply:
So it should be 🤦🏻